examples/demo_acl.pp

```diff
@@ -16,29 +16,48 @@
 
 class ciscopuppet::demo_acl {
   cisco_acl { 'ipv4 my_ipv4_acl':
-    ensure                 => 'present'
-    stats_per_entry        => false,
-    fragments              => 'permit'
+    ensure          => 'present',
+    stats_per_entry => false,
+    fragments       => 'permit'
   }
   cisco_ace { 'ipv4 my_ipv4_acl 10':
-    ensure                 => 'present',
-    action                 => 'permit',
-    proto                  => 'tcp',
-    src_addr               => '1.2.3.4 2.3.4.5',
-    src_port               => 'eq 40',
-    dst_addr               => '8.9.0.4/32',
-    dst_port               => 'range 32 56',
+    ensur    => 'present',
```

Comment:
I kept playing around with the demo_acl on the master for testing. Sorry about this typo. I'll fix and test again.